docs/app/docs/faq.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Frequently Asked Questions
+description: Frequently Asked Questions about Devbox
+---
+
+This doc contains answers to frequently asked questions about Devbox that are not covered elsewhere in our documentation. If you have a question that isn't covered here, feel free to ask us on our [Discord](https://discord.gg/jetpack-io), or [open an issue](https://github.com/jetpack-io/devbox/issues) on our GitHub repository.
+
+## How does Devbox work?
+
+Devbox generates isolated, reproducible development environments using the [Nix package manager](https://nixos.org/). Devbox uses Nix to install packages, and then creates an isolated shell environment for your project by symlinking the packages you need into your project directory.
+
+## Where does Devbox install my packages?
+
+Devbox and Nix install your packages in the read-only Nix store, usually located at `/nix/store`. Devbox then creates your environment by symlinking the packages you need into the `.devbox` directory in your project.
+
+## How do I clean up unused packages from the Nix Store?
+
+You can use `nix store gc` to automatically clean up packages that are no longer needed for your projects.
```

Comment:
Sometimes nix is not in the path. We can also tell them to do `devbox run — nix store gc`?

Reply:
Maybe i'll just default it to that, seems tidier